Unix/installbuilder/datafiles/AIX.data

```diff
@@ -74,3 +87,15 @@ ${{SERVICE_SCRIPT}} start
 #include OmiService_funcs
 StopOmiService
 RemoveOmiService
+
+%Postuninstall_100
+egrep "^omi:" /etc/passwd 2>&1 >/dev/null
+if [ $? -eq 0 ]; then
+    echo "Deleting omi service account ..."
+    userdel omi
+fi
+egrep "^omi:" /etc/group 2>&1 >/dev/null
+if [ $? -eq 0 ]; then
+    echo "Deleting omi group ..."
+    rmgroup omi
+fi
```

Comment:
This may or may not be a problem here. On Linux, this would **definitely** be a problem on upgrade, as post uninstall for the *old version* can run *AFTER* post install for the new version.

We need to verify, one way or the other, for AIX, HP, and Solaris. This needs to be done before signoff.